other/deprecating-apps.md

```diff
@@ -0,0 +1,12 @@
+# Deprecating Apps
+
+NYPL creates different applications for user interfaces, data pipeline flow, APIs, and more. While we maintain these applications, sometimes it's better to move from one platform to another for many reasons.
+
+When this happens, the developer(s) working on the new iteration of an application MUST let others know that the older version will be deprecated. This can reduce confusion on whether packages need to be updated for an app or codebase.
+
+The above also applies to codebases which will no longer be used, such as moving code from one codebase to another.
+
+Developer(s) MUST:
+* update the codebase's README with a deprecated and a no longer supported note
+* let the team know that an app/codebase is deprecated
+* update related documentation where the app/codebase's name comes up
```

Comment:
Not sure where we would find these unless there was more information on these pages regarding a projects interdependencies and documentation. We should probably work together on this to make sure we can add the appropriate links, etc. per project. Not sure how much we need to add here since this is a public repository.

Reply:
Right, this is specific to each repo and maybe some don't have any interdependencies.